src/main/java/org/thoughtcrime/securesms/components/emoji/AutoScaledEmojiTextView.java

```diff
@@ -45,6 +45,16 @@ public void setText(@Nullable CharSequence text, BufferType type) {
     super.setText(text, type);
   }
 
+  @Override
+  public void setTextSize(float size) {
+    setTextSize(TypedValue.COMPLEX_UNIT_SP, size);
+  }
+
+  @Override
+  public void setTextSize(int unit, float size) {
+    originalFontSize = TypedValue.applyDimension(unit, size, getResources().getDisplayMetrics());
+    super.setTextSize(unit, size);
+  }
```

Comment:
why is that needed now?

Reply:
re-review, I just fixed it instead of removing it, for some user with Samsung 12 the system setting was not taking effect in the text size and I suspect it is because this is necessary anyways to refresh the font size, since the saved value is in PX, not SP